torchao/quantization/quant_api.py

```diff
@@ -409,7 +409,22 @@ def apply_int4_weight_only_quant(weight):
         zero_point_dtype = torch.bfloat16
         zero_point_domain = ZeroPointDomain.FLOAT
         layout_type = TensorCoreTiledLayoutType(inner_k_tiles=inner_k_tiles)
-        return to_affine_quantized(weight, mapping_type, block_size, target_dtype, quant_min, quant_max, eps, zero_point_dtype=zero_point_dtype, preserve_zero=preserve_zero, zero_point_domain=zero_point_domain, layout_type=layout_type)
+
+        if(use_hqq):
+            return  AffineQuantizedTensor.from_float(
```

Comment:
and here

Reply:
https://github.com/pytorch/ao/pull/605/commits/3303d95522b8750d81b13e074ece3677890050b0